aeron-client/src/main/java/io/aeron/ChannelUriStringBuilder.java

```diff
@@ -1228,6 +1228,36 @@ public ChannelUriStringBuilder taggedFlowControl(
         return flowControl(flowControlStr);
     }
 
+
+    /**
+     * Set the min flow control settings to be used on channel.  All specified values may be null and the default
+     * specified in the MediaDriver.Context will be used instead.
+     *
+     * @param groupSize group size required to allow publications for this channel to be move to connected status.
+     * @param timeout timeout receivers, default is ns, but allows suffixing of time units (e.g. 100ms).
+     * @return this for fluent API.
+     */
+    public ChannelUriStringBuilder minFlowControl(
```

Comment:
Should this also be done for tagged?

Reply:
Yes :-), https://github.com/real-logic/aeron/blob/master/aeron-client/src/main/java/io/aeron/ChannelUriStringBuilder.java#L1201